teamB/junhyungcho/models/likeDao.js

```diff
@@ -0,0 +1,43 @@
+const { DataSource } = require('typeorm');
+
+const database = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE
+})
+
+database
+    .initialize()
+    .then(() => {
+      console.log("Like data Source has been initialized!")
+    })
+    .catch((err) => {
+      console.error("Error during Data Source initialization", err)
+      appDataSource.destroy()
+    })
+    .finally(() => console.log("==========================================="));
+
+const pushHeart = async ( userId, postId ) => {
```

Comment:
데이터를 생성하는 것이므로 다음과 같이 이름을 수정하면 조금 더 좋을 것 같습니다.
```suggestion
const createHeart = async ( userId, postId ) => {
```